### ISSUE TITLE
Extra field DNSSEC

### ISSUE THREAD

**Issue by robinbzk** [internetstandards/Internet.nl-dashboard#262]
![image](https://user-images.githubusercontent.com/28256969/109287046-5cbcd600-7823-11eb-94d6-b1f9f8c72634.png)
Could probably be simplified by using the Category value of DNSSEC.

**Comment by stitch** [comment]
That's true, and requires a code change. It currently works and gives the correct value, so let's keep the code as it is.
